superset/assets/javascripts/SqlLab/components/SplitPane.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```

Comment:
I don't think this file belongs in this PR

Reply:
@mistercrunch i have removed this from the branch and pr